src/YQPkgDescriptionView.cc

```diff
@@ -312,9 +313,11 @@ void YQPkgDescriptionView::initLang()
         _langWithCountry = lang_cstr;
 
          // remove .utf8 / @euro etc.
-        _langWithCountry.replace( QRegularExpression( "[@\\.].*$" ), "" );
+        static QRegularExpression reCountry( "[@\\.].*$" );
```

Comment:
This is pointless: That function is called just one single time during the whole lifetime of the program; only in the constructor of this YQPkgDesciptionView which is created exactly once ever.

That makes those static variables even counterproductive; they would store something that is never reused.